sequencing/variant_annotator/dxapp.json

```diff
@@ -91,29 +120,30 @@
       "optional" : true,
       "patterns": [
         "*.tbi"
-      ]
+      ],
+      "help": "(Optional) An array of index files that match the site-only variant files."
     }
   ],
   "runSpec": {
-  "execDepends": [
-      {
-        "name": "pigz"
-      },
-      {
-        "name": "tabix"
+    "distribution": "Ubuntu",
+    "release": "14.04",
+    "interpreter": "python2.7",
+    "file": "src/variant_annotator.py",
+    "timeoutPolicy": {
+      "*": {
+        "hours": 24
       }
-    ],
-    "interpreter": "bash",
-    "file": "src/variant_annotator.sh"
+    }
   },
-  "resources" : "project-BYpFk1Q0pB0xzQY8ZxgJFv1V",
-  "access": {
-    "network": [
-      "*"
-    ],
-     "allProjects": "VIEW"
+  "regionalOptions": {
+    "aws:us-east-1": {"systemRequirements": {"*": {"instanceType": "mem1_ssd1_x8"}},
+                      "assetDepends": [{"id": "record-FJ1pP980FJfZFzqYG88B8Zj1"}]},
+    "azure:westus": {"systemRequirements": {"*": {"instanceType": "azure:mem1_ssd1_x8"}},
+                     "assetDepends": [{"id": "record-FJ1vJgQ99qgf9kQk264zVpKv"}]}
   },
-  "developers": ["user-tnperson", "user-auv13"],
   "openSource" : true,
-  "authorizedUsers" : ["org-geisinger_affiliate", "org-geisinger", "org-support", "user-codushlaine", "user-semanti.mukherjee", "user-alexander.li", "user-shannon.bruse", "user-jan.freudenberg", "user-crisvh"]
+  "details": {
+    "upstreamUrl": "https://software.broadinstitute.org/gatk/documentation/tooldocs/3.8-0/org_broadinstitute_gatk_tools_walkers_annotator_VariantAnnotator.php",
+    "whatsNew": "* 1.0.0 First version of the VariantAnnotator app."
```

Comment:
"* 1.0.0 First version of the VariantAnnotator app." -> "* 1.0.0 First version of the GATK3 VariantAnnotator app."

Reply:
Done